src/xdocs/config_coding.xml

```diff
@@ -3153,13 +3153,42 @@ public void foo(int i, String s) {}
         </p>
```

Comment:
See this paragraph section. We explain reasoning for no package defined, should we extend this for validating the directory structure too?